.github/workflows/appsignals-e2e-eks-test.yml

```diff
@@ -219,6 +220,25 @@ jobs:
             fi
           done
 
+      - name: Install AppSignals addon
+        id: install-addon
+        if: steps.deploy-sample-app.outcome == 'success' && !cancelled()
+        run: |
+          source ${{ env.TEST_RESOURCES_FOLDER }}/.github/workflows/util/execute_and_retry.sh
+          execute_and_retry 2 \
+          "${{ env.TEST_RESOURCES_FOLDER }}/enablement-script/scripts/eks/appsignals/enable-app-signals.sh \
+          ${{ inputs.test-cluster-name }} \
+          ${{ inputs.aws-region }} \
+          ${{ env.SAMPLE_APP_NAMESPACE }}" \
+          "${{ env.TEST_RESOURCES_FOLDER }}/enablement-script/scripts/eks/appsignals/clean-app-signals.sh \
+          ${{ inputs.test-cluster-name }} \
+          ${{ inputs.aws-region }} \
+          ${{ env.SAMPLE_APP_NAMESPACE }} && \
+          aws eks update-kubeconfig --name ${{ inputs.test-cluster-name }} --region ${{ inputs.aws-region }}"
```

Comment:
You can run this step from the directory instead of pointing to it each time. That or make the improvement to set this as a reusable action for the future.

Reply:
Done